src/scss/main.scss

```diff
@@ -10,6 +10,7 @@
 // Layout
 
 @import 'layout/header';
+@import 'layout/teamSection';
```

Comment:
don't use camelCase notation on filenames. Please rename it to `team-section.scss`. Same for the template.

Reply:
Please update this line to @import 'layout/team-section';
and please, rename the template to teamSection.hbs. In this case as the content type ID in contentful is with camelCase, handlebars will show the same name for the tamplate. sorry about the confusion ! :(

Reply:
Got it. Will do :)